packages/harmony/src/components/button/SocialButton/SocialButton.tsx

```diff
@@ -48,13 +46,26 @@ const getSocialButtonProps = (type: SocialMedia): ButtonProps => {
 }
 
 export const SocialButton = (props: SocialButtonProps) => {
-  const { socialType, className, ...rest } = props
+  const { socialType, ...rest } = props
   const socialButtonProps = getSocialButtonProps(socialType)
+
+  const instagramBackgroundCss: CSSObject = {
+    background: `radial-gradient(61.46% 59.09% at 36.25% 96.55%, #ffd600 0%, #ff6930 48.44%, #fe3b36 73.44%, rgba(254, 59, 54, 0) 100%),
+      radial-gradient(202.83% 136.37% at 84.5% 113.5%, #ff1b90 24.39%, #f80261 43.67%, #ed00c0 68.85%, #c500e9 77.68%, #7017ff 89.32%)`
+  }
+
   return (
     <Button
       {...socialButtonProps}
       {...rest}
-      className={cn(styles.root, socialButtonProps.className, className)}
+      css={({ color }) => ({
+        border: 'transparent',
+        ...(socialType === 'instagram' && instagramBackgroundCss),
+
+        '& svg path': {
```

Comment:
we were having trouble with `&` in electron, right?

probably different now that it's going through emotion, but flagging just in case

Reply:
yea emotion is handling this